_pytest/3296.trivial

```diff
@@ -0,0 +1 @@
+unify cli verbosity handling
```

Comment:
I would write something that makes sense to users:

```
Refactoring to unify how verbosity is handled internally.
```

And we also need a new `feature` entry:

```rst
New ``--verbosity`` flag to set verbosity level explicitly.
```